src/linagora.esn.calendar/app/calendar-configuration/calendar-configuration-tab-main/calendar-configuration-tab-main.controller.js

```diff
@@ -1,4 +1,14 @@
-(function() {
+require('esn-frontend-common-libs/src/frontend/js/modules/session.js');
+require('esn-frontend-common-libs/src/frontend/js/modules/user/user-utils.service.js');
```

Comment:
I will not go further on the review since it is already here:

You have to require external modules from `esn-frontend-common-libs` from the module file definition ie `frontend/app/app.js`